### PR TITLE
docs(user): updates doc osp partner perspective

### PR DESCRIPTION
**Pull request by Leon-Gr**
## Description

Added documentation for partners registered via an OSP

#255

## Why

This Documentation can help out users, who were onboarded with an OSP

### CODE REVIEW

**Review by evegufy — COMMENTED** [review]
looks good in general but the file is currently located directly under docs/user/01. Onboarding, is that supposed to the location?

File: docs/user/01. Onboarding/OSPDrivenOnboarding.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## Users invited via OSP
+
+This part of the documentation will help users, who were registered via an Onboarding Service Provider (OSP) to get along. Please note, that this documentation only starts at the time you were successfully registered with your OSP and received your welcome mail from the portal. Any help or guidance regarding the registration on any site of your OSP, needs to be obtained from your OSP. Please refer to [Onboarding Service Provider](/docs/developer/00.%20Personas/05.%20Onboarding%20Service%20Provider.md) if you need more information about OSPs in general.
```
Comment (jjeroch) [inline docs/user/01. Onboarding/OSPDrivenOnboarding.md:3]: @evegufy  can you quickly check if that will actually work? Linking to a developer doc docu. Will this successfully work in our help app? 
@Leon-Gr  I think we should add a own OSP Persona inside the user documentation describing the OSP from a pure functional side

File: docs/user/01. Onboarding/OSPDrivenOnboarding.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+### Welcome Mail & First Login
+
+As soon as the registration process is done on your OSP side, you will receive a welcome mail, that contains information on how you can proceed. It will provide you with a link and information about how to log in to the dataspace.
```
Comment (jjeroch) [inline docs/user/01. Onboarding/OSPDrivenOnboarding.md:7]: I suggest to include additional information (as we have it inside the email) that due to the OSP link the login is supposed to take place via the email mentioned login account and with the personal user credentials which the user does know already from the OSP network login

File: docs/user/01. Onboarding/OSPDrivenOnboarding.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+### T&Cs
+
+When you first log in, you will be prompted with an overlay that asks you to consent to the terms and conditions. As soon as this is done the overlay will close and you can start browsing the site.
```
Comment (jjeroch) [inline docs/user/01. Onboarding/OSPDrivenOnboarding.md:11]: We should mention that the the terms and conditions may vary based on the company assigned role (given by the OSP registration creation).

File: docs/user/01. Onboarding/OSPDrivenOnboarding.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+## Users invited via OSP
+
+This part of the documentation will help users, who were registered via an Onboarding Service Provider (OSP) to get along. Please note, that this documentation only starts at the time you were successfully registered with your OSP and received your welcome mail from the portal. Any help or guidance regarding the registration on any site of your OSP, needs to be obtained from your OSP. Please refer to [Onboarding Service Provider](/docs/developer/00.%20Personas/05.%20Onboarding%20Service%20Provider.md) if you need more information about OSPs in general.
+
+### Welcome Mail & First Login
```
Comment (jjeroch) [inline docs/user/01. Onboarding/OSPDrivenOnboarding.md:5]: I miss anything about declining my registration - isnt that supported yet?

File: docs/user/01. Onboarding/OSPDrivenOnboarding.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+### T&Cs
+
+When you first log in, you will be prompted with an overlay that asks you to consent to the terms and conditions. As soon as this is done the overlay will close and you can start browsing the site.
+<br>
+**Please Note:** At this point you will not yet have access to the full functionality, this will only be possible once your registration is finally approved (see next paragraph).
+
+### Status Overlay
+
+You will then see an overlay that tells you about the status of your registration. The individual steps, that need to be checked and done prior to your final registration will be listed here. You can close it at any time and can reopen it with the bar at the top. The required steps are the same as they are for a regular operator triggered registration. That also includes the generation of the BPNL.
```
Comment (jjeroch) [inline docs/user/01. Onboarding/OSPDrivenOnboarding.md:17]: Missing: as soon as all steps are successfully performed; an automatic email will get send out

**Review by jjeroch — CHANGES_REQUESTED** [review]

File: docs/user/01. Onboarding/OSPDrivenOnboarding.md (hunk 1 above)
Comment (evegufy) [reply docs/user/01. Onboarding/OSPDrivenOnboarding.md:3]: @jjeroch just checked the link for Audit Log Details in [here](https://raw.githubusercontent.com/eclipse-tractusx/portal-assets/main/docs/user/03.%20User%20Management/02.%20Modify%20User%20Account/06.%20Delete%20User.md) on INT, it works

File: docs/user/01. Onboarding/OSPDrivenOnboarding.md (hunk 4 above)
Comment (jjeroch) [reply docs/user/01. Onboarding/OSPDrivenOnboarding.md:5]: danke @Leon-Gr

File: docs/user/01. Onboarding/OSPDrivenOnboarding.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+### Status Overlay
+
+You will then see an overlay that tells you about the status of your registration. The individual steps, that need to be checked and done prior to your final registration will be listed here. You can close it at any time and can reopen it with the bar at the top. The required steps are the same as they are for a regular operator triggered registration. That also includes the generation of the BPNL. As soon as all steps have been finished successfully, you will receive an status information mail.
+<br>
+**Please Note:** you do not need to perform anything here, these are automatic steps, that generally do not require your active participation at that point.
+
+### Decline Registration
+
+In case you received the earlier mentioned welcome mail and you do not want to part of the dataspace, you can scroll down to the bottom of the page and click the link for declining the registration.
```
[the remark this answers is not in the document]
Comment (Leon-Gr) [reply docs/user/01. Onboarding/OSPDrivenOnboarding.md:23]: Done

**Review by jjeroch — APPROVED** [review]

File: docs/user/01. Onboarding/04. OSP/01. Oboarding with an OSP.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
+## Users invited via OSP
+
+This part of the documentation will help users, who were registered via an Onboarding Service Provider (OSP) to get along. Please note, that this documentation only starts at the time you were successfully registered with your OSP and received your welcome mail from the portal. Any help or guidance regarding the registration on any site of your OSP, needs to be obtained from your OSP. Please refer to [Onboarding Service Provider](/docs/developer/00.%20Personas/05.%20Onboarding%20Service%20Provider.md) if you need more information about OSPs in general.
+
+### Welcome Mail & First Login
+
+As soon as the registration process is done on your OSP side, you will receive a welcome mail, that contains information on how you can proceed. It will provide you with a link and information about how to log in to the dataspace. For the login you can use the account you're using with your OSP party already and entering your dataspace user credentials. In case the authentication runs on errors, please have a look at the [help](https://portal.int.demo.catena-x.net/documentation/?path=docs%2F09.+Others%28s%29%2F01.+Login.md) or contact your Onboarding Service Provider.
```
Comment (evegufy) [inline docs/user/01. Onboarding/04. OSP/01. Oboarding with an OSP.md:7]: please remove reference to int env

**Review by evegufy — CHANGES_REQUESTED** [review]
looks good so far, just one additional note and please also update the index.md

**Review by evegufy — APPROVED** [review]

**Pull request merged**